examples/plotting/plot_grid_data_overlay.py

```diff
@@ -0,0 +1,80 @@
+"""
+=====================================
+Create a grid plot with data overlay
+=====================================
+
+An example which creates a plot of a gridded NEXRAD radar on a map
+with latitude and NCEP North American regional reanalysis (NARR) pressure
+ is plotted on top of the grid.
```

Comment:
no space at the beginning of this line, it causes a warning when building the docs.